### ISSUE TITLE
LinkError: post-link script failed for package conda-forge::mss-1.7.3-py36_0

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#365]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/365) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Fresh miniconda3 new environment and found an issue with the linux post-link script

```
#!text

Proceed ([y]/n)? y

Preparing transaction: done
Verifying transaction: done
Executing transaction: failed
ERROR conda.core.link:_execute(502): An error occurred while installing package 'conda-forge::mss-1.7.3-py36_0'.
LinkError: post-link script failed for package conda-forge::mss-1.7.3-py36_0
running your command again with `-v` will provide additional information
location of failed script: /home/user/miniconda3/envs/mssenv/bin/.mss-post-link.sh
==> script messages <==
<None>

Attempting to roll back.

Rolling back transaction: done

LinkError: post-link script failed for package conda-forge::mss-1.7.3-py36_0
running your command again with `-v` will provide additional information
location of failed script: /home/user/miniconda3/envs/mssenv/bin/.mss-post-link.sh
==> script messages <==
<None>
```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

This has to become added on conda-forge before we can verify it 

```
#!text

diff --git a/post-link.sh b/post-link.sh
index 11caf2f..daf334c 100755
--- a/post-link.sh
+++ b/post-link.sh
@@ -1,7 +1,9 @@
 unamestr=`uname`
 # only if we don't execute in a docker environment
 if [[ "$(cat /proc/1/sched | head -n 1)" = "systemd (1, #threads: 1)" ]] && [[ "$unamestr" == 'Linux' ]]; then
-    mss -m
-else 
-       echo "called in a container"
+    msscmd="$CONDA_PREFIX/bin/mss"
+    $msscmd -m 
+    echo "menue entry ($msscmd -m): done"
+else
+    echo "called in a container"
 fi

```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

call of script fixed by conda dir, fixes #365
